compress/compress_private.h

```diff
@@ -24,12 +24,12 @@
 #define MUTT_COMPRESS_COMPRESS_PRIVATE_H
 
 #define COMPRESS_OPS(_name)                  \
-  const struct ComprOps compr_##_name##_ops = {     \
-    .name       = #_name,                           \
-    .open       = compr_##_name##_open,             \
-    .compress   = compr_##_name##_compress,         \
-    .decompress = compr_##_name##_decompress,       \
-    .close      = compr_##_name##_close,            \
+  const struct ComprOps compr_##_name##_ops = {        \
+    .name            = #_name,                         \
+    .open            = compr_##_name##_open,           \
+    .compress        = compr_##_name##_compress,       \
+    .decompress      = compr_##_name##_decompress,     \
+    .close           = compr_##_name##_close,          \
   };
```

Comment:
I guess the changes to this file can now be undone

Reply:
Yes, of cause ;)